addons/mail/tools/discuss.py

```diff
@@ -78,6 +78,7 @@ def get_sfu_key(env) -> str | None:
     }
 )
 
+NO_VALUE = object()
```

Comment:
Note: `None` cannot be the default value as it can be an actually given value and it needs to be differenciated